src/translate/pddl_to_prolog.py

```diff
@@ -142,6 +142,9 @@ def __str__(self):
         cond_str = ", ".join(map(str, self.conditions))
         return "%s :- %s." % (self.effect, cond_str)
 
+def get_definition_fluent(pne: pddl.PrimitiveNumericExpression):
+    return pddl.Atom(f"@def-{pne.symbol}", pne.args)
+
```

Comment:
- I think Gabi mentioned this, but I think it could indeed be nicer to move this to `normalize.py` and drop the `import pddl_to_prolog` there.
- I would make the parameter name match the one chosen in `normalize.py`.
- The name `get_definition_fluent` could maybe be less generic but I don't have a good suggestion.

Reply:
Parameter names now match. I moved the function to `normalize.py` and renamed it to `get_pne_definition_predicate`. I am not totally happy about the predicate part but it's the analogous name to the analogous functions `get_action_predicate` and `get_axiom_predicate` just next to it.